tests/framework/prometheus.go

```diff
@@ -115,19 +112,13 @@ func (f *Framework) getPrometheusURL() string {
 	if utils.IsOpenshift(f.K8sClient) {
 		gomega.Eventually(func() bool {
 			host, err = f.runOcCommand("-n", "openshift-monitoring", "get", "route", "prometheus-k8s", "--template", "{{.spec.host}}")
-			if err != nil {
-				return false
-			}
-			return true
+			return err != nil
```

Comment:
So I see some failures on the destructive lane, probably because this got inversed
(note we return false when err != nil unlike everywhere else)